src/main/java/com/hiqes/andele/RequestOwnerSupportAppCompatActivity.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2015 HIQES LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hiqes.andele;
+
+import android.annotation.TargetApi;
+import android.app.Application;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import java.lang.ref.WeakReference;
+
+class RequestOwnerSupportAppCompatActivity extends RequestOwner {
```

Comment:
This should be the new class for Android X app compat, the existing one should have been left as-is.

Reply:
Yes, Done.